tests/cases/fourslash/uncommentSelection1.ts

```diff
@@ -0,0 +1,38 @@
+// Simple comment selection cases.
+
+//// //let var1[| = 1;

```

Comment:
If I uncomment `//// stuff`, I get `// stuff`, right?

Reply:
Correct, I have added the test to be sure.